internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -576,7 +576,9 @@ func (cf CloudFormation) errOnFailedStack(stackName string) error {
 	}
 	status := aws.StringValue(stack.StackStatus)
 	if cloudformation.StackStatus(status).IsFailure() {
-		return fmt.Errorf("stack %s did not complete successfully and exited with status %s", stackName, status)
+		return fmt.Errorf("stack %s did not complete successfully and exited with status %s.\n"+
+			"You can retry building your service by updating the manifest (if needed) and running "+
```

Comment:
is there a way we could print this _only_ if the app runner resource fails? for example, it probably doesn't make sense to print a message telling someone to rebuild their service if the failure was due to a misconfigured RDS addon or something

Reply:
I'm not sure if we can see what is the exact error here (at least for now). We could maybe update the statement to be something like "Please see the error above or update the manifest(if needed)". By doing this, if we have an error which is "null" (current scenario with AR services) the user would want to look at their manifest. In all other cases, we should see a meaningful actionable error so we should be good for them in any case.

Reply:
+1 to @dannyrandall ! A sort of related & minor concern that I have is that, wordings like "(if needed)" is quite ambiguous. Like we've discussed before, this message is mostly useful for people that has very little experience with Copilot or AWS. I'm worried that for this group of audience, seeing "if needed" would just add to their frustration - like "how do I know if it's needed?"

Reply:
That does make sense. Do we have any suggestions on what the wording should be? Is this better: "You can retry building your service by fixing the error shown above or by double-checking your service's manifest and running copilot svc deploy again."

Reply:
What about something like this:
> "You may fix the error by updating the service code or the configuration in the manifest."

Reply:
I like it. So then is this okay:

>You may fix the error by updating the service code or the configuration in the manifest. You can then retry building your service by running copilot svc deploy.